### PR TITLE
Add merchant context param

### PR DESCRIPTION
**Pull request by moriaam**

### CODE REVIEW

File: integration.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ Recurring payments offer more versatility in the way payments are collected from
 Assuming you've already included the SDK itself on your page, you now need to place the recurring payments form somewhere. This form will allow users to register for recurring payments:
 
 ```html
-<form action="/subscription" method="POST" class="klarna-form" data-api-key="test_d4ca9ed6-489b-42b3-8011-dacdcee0fdb6" data-flow="recurring-purchase" data-locale="sv">
+<form action="/subscription" method="POST" class="klarna-form" data-api-key="test_d4ca9ed6-489b-42b3-8011-dacdcee0fdb6" data-flow="recurring-purchase" data-locale="sv" data-merchant-context="daily pass">
```
Comment (danevron) [inline integration.md]: Maybe it's better to use subscription here.

File: integration.md [hunk 2]
```diff
@@ -220,3 +220,4 @@ data-flow | The type of on-demand form to display. The supported values are `pur
 data-locale | The locale in which the form should be presented.
 data-amount | This attribute is valid only for purchase forms and denotes the purchase's total price. The value should be multiplied by 100, so if your price is "9,90" it should be sent as "990".
 data-currency | This attribute is valid only for purchase forms and denotes the purchase's currency, as an [ISO 4217 code](https://en.wikipedia.org/wiki/ISO_4217).
+data-merchant-context | This attribute is used to track conversion rate in Google Analytics based on any given attribute, i.e. "daily pass".
```
Comment (danevron) [inline integration.md]: I suggest writing that this parameter is optional (maybe another column) and we recommend sending information about the product. Klarna will use this to segment your traffic and will be able to offer ideas that may help your conversion as a merchant. A practical example is to divide the traffic between "daily pass" and "weekly pass".

And please talk to Tomer about it.

File: integration.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ While the specifics vary slightly depending on your use case, there are three ma
 2. Place a form on the page and mark it with the `klarna-form` class. While we don't want to dive into the specifics just yet, here is an example of such a form:
 
    ```html
-   <form action="/some_action" method="POST" class="klarna-form" data-api-key="test_d4ca9ed6-489b-42b3-8011-dacdcee0fdb6" data-flow="registration" data-locale="sv">
+   <form action="/some_action" method="POST" class="klarna-form" data-api-key="test_d4ca9ed6-489b-42b3-8011-dacdcee0fdb6" data-flow="registration" data-locale="sv" data-merchant-context="daily pass">
```
Comment (danevron) [inline integration.md]: Can you change the flow parameter as we don't support registration?

File: integration.md (hunk 1 above)
Comment (moriaam) [inline integration.md]: done

File: integration.md (hunk 2 above)
Comment (moriaam) [inline integration.md]: done

File: integration.md (hunk 3 above)
Comment (moriaam) [inline integration.md]: done

**Pull request merged**